partitura/utils/music.py

```diff
@@ -637,19 +732,26 @@ def estimate_clef_properties(pitches):
     center = np.median(pitches)
     # number, sign, line, octave_change):
     # clefs = [score.Clef(1, 'F', 4, 0), score.Clef(1, 'G', 2, 0)]
-    clefs = [dict(sign='F', line=4, octave_change=0),
-             dict(sign='G', line=2, octave_change=0)]
-    f = interp1d([0, 49, 70, 127], [0, 0, 1, 1], kind='nearest')
+    clefs = [
+        dict(sign="F", line=4, octave_change=0),
+        dict(sign="G", line=2, octave_change=0),
+    ]
+    f = interp1d([0, 49, 70, 127], [0, 0, 1, 1], kind="nearest")
     return clefs[int(f(center))]
 
 
-def compute_pianoroll(note_info, time_unit='auto', time_div='auto',
```

Comment:
As in similar comments, I slightly prefer the old formatting of definitions, but I am ok with the new one if that is preferred by PEP8.